### PR TITLE
Feature/websites 18

### PR DESCRIPTION
**Pull request by YascaraMendozaLopez**
In this pull request, the detail and editing of E-Commerce opinions was added

### CODE REVIEW

**Comment by olmedo90** [comment]
image

It is according to the figma. continue with the same interest. 👍

**Comment by YascaraMendozaLopez** [comment]
@Ditmar  Engineer, could you please review my PR

File: src/Theme/Theme.js [hunk 1]
```diff
@@ -45,13 +45,44 @@ const AllertaStencilBold = {
   fontWeight: 700,
   src: `url(${AllertaStencil})`,
 };
+const DMSansRegular ={
+  fontFamily: "DMSans-Regular",
```
Comment (Ditmar) [inline src/Theme/Theme.js]: please use simple quotes

File: src/Theme/Theme.js [hunk 2]
```diff
@@ -45,13 +45,44 @@ const AllertaStencilBold = {
...
+  fontStyle: "normal",
```
Comment (Ditmar) [inline src/Theme/Theme.js]: please use simple quotes

File: src/ui/components/opinions/ApproveDelete/ApproveDelete.style.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        display: "flex",
+        justifyContent: "right",
+        padding: "13px",
+    },
+    icon: {
+        display: "flex",
+        justifyContent: "center",
+    },
+    text: {
+        fontFamily: "DMSans-Regular",
```
Comment (Ditmar) [inline src/ui/components/opinions/ApproveDelete/ApproveDelete.style.js]: please use simple quotes

File: src/ui/components/opinions/OpinionDetail/OpinionDetail.styles.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+
+    },
+    product: {
+        display:"flex",
+        flexDirection: "column",
+
+    },
+    productDetail:{
+        display: "grid",
```
Comment (Ditmar) [inline src/ui/components/opinions/OpinionDetail/OpinionDetail.styles.js]: please use simple quotes

**Review by Ditmar — APPROVED** [review]
really nice work approved please only chage the doble quote to simple quote

**Comment by YascaraMendozaLopez** [comment]
@Ditmar Engineer, I already resolved the conflict again

**Review by Ditmar — APPROVED** [review]
approved

**Pull request merged**